ws-client/src/helpers.rs

```diff
@@ -181,6 +181,20 @@ pub async fn stop_subscription(sender: &mut WsSender, manager: &mut RequestManag
 	}
 }
 
+/// Garbage collect all active subscriptions.
+pub async fn close_all_subscriptions(sender: &mut WsSender, mut manager: RequestManager) {
+	let subs = manager.subscriptions();
```

Comment:
we could maybe batch these instead but the code will become event uglier w.r.t borrowing issues.